pipelines/ceus/ceus.py

```diff
@@ -127,7 +127,7 @@ def generate_result_plots(self):
         base_enduses.remove('daytype')
         base_enduses.remove('Heating')
         base_enduses.remove('Cooling')
-        ticks = np.arange(0, 25, 3) 
+        ticks = np.arange(0, 25, 3)
```

Comment:
This is specified in two different places with the same value. Specify once as a class prop.

Reply:
fixed